store.html

```diff
@@ -0,0 +1,110 @@
+    
+                    <div>
+                        <label for="surname">Last name*</label>
+                        <input id="surname" type="text">
+                    </div>
+                </div>
+
+                <div class="two">
+                <label for="address">Address*</label>
+                <input id="address" type="text">
+                </div>
+
+                <div class="three">
+                <label for="address2">Address 2</label>
+                <input id="address2" type="text">
+                </div>
+
+                <div class="four">
+                    <div>
+                        <label for="city">City*</label >
+                        <select id="city" name="city" required>
+                            <option value=" " disabled selected hidden>Select your city</option>
+                            <option value="london">london</option> 
+                            <option value="manchester">Manchester</option>
+                            <option value="Birmigan">Birmigan</option>
+                            <option value="glasgow">Glasgow</option>
+                            <option value="ireland">Ireland</option>
+                        </select>
+                    </div>
+    
+                    <div>
+                        <label for="pc">Post Code*</label>
+                        <input id="pc" type="text">
+                    </div>
+                </div>
+
+                <div class="five">
+                    <fieldset>
+                        <legend>select a color</legend>
+    
+                        <input id="selectcolor" type="radio" name="color" checked>
+                        <label for="selectcolor">Karma orange</label>
+    
+                        <input id="selectcolor" type="radio" name="color">
+                        <label for="selectcolor">Space grey</label>
+                    </fieldset>
+                </div>
+
+                <div class="six">
+                    <input id="terms"  type="checkbox" class="checkbox" checked required>
+                    <label for="terms">By placing yosur order you agree to Karma's <a href="#">Terms & Conditions</a>. *</label>
+                </div>
+                <div class="seven">
+                    <input  type="submit" value="Place my order">
+                </div>
+            </form>
+        </section>
+
+        <div class="photo">
+
+        </div>
+
+    </div>
+
+
+    <hr>
+    <footer>
+        <p class="Join">Join us on</p>
+          <ul>
+            <li><a href="https://twitter.com/"><img src="/img/twitter-icon.svg" alt="twitterLogo"></a></li>
+            <li><a href="https://www.facebook.com/"><img src="/img/facebook-icon.svg" alt="facebookLogo"></a></li>
+            <li><a href="https://www.instagram.com/"><img src="/img/instagram-icon.svg" alt="instagramLogo"></a></li>
+          </ul>
+      <p>&#169Karma Mobility,Inc.</p>
+      </footer>
+</body>
+</html>
```

Comment:
[code-style] - new line in the end of the file

Reply:
thanks for all the comments, it's all fixed.